.netlifyredirects

```diff
@@ -5,4 +5,5 @@
 /documentation/*   https://guides.emberjs.com/release/:splat
 /builds/*   /releases/:splat
 /about*     /
+/about/*    /:splat
```

Comment:
No this is the wrong change, it should be 

```suggestion
/legal    /about/legal
```
You don't need to worry about the fact that there is an existing redirect that seems like it might apply, you can read up on the functionality of Netlify here: https://www.netlify.com/docs/redirects/#note-on-shadowing